master/internal/api_tasks.go

```diff
@@ -68,7 +68,12 @@ func expFromAllocationID(
 }
 
 func canAccessNTSCTask(ctx context.Context, curUser model.User, taskID model.TaskID) (bool, error) {
-	taskOwnerID, err := db.GetCommandOwnerID(ctx, taskID)
+	snapshot := command.CommandSnapshot{}
+	err := db.Bun().NewSelect().Model(&snapshot).
```

Comment:
nothing wrong with this, but I feel like it would be more elegant to abstract the snapshot retrieval somewhere in the db package.

Reply:
I updated it to this. we didn't need the whole snapshot https://github.com/determined-ai/determined/pull/5666/files#diff-ab0efb51a96733e18bae6be6a87fd0e0162868d9d308bf474984baf1fb2de8aaR138